.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "extern/openpgp-api-lib"]
 	path = extern/openpgp-api-lib
-	url = https://github.com/open-keychain/openpgp-api.git
+	url = https://github.com/hannesa2/openpgp-api.git
```

Comment:
Please merge the PR there and I can switch back to https://github.com/open-keychain/openpgp-api.git